arm/Microsoft.Insights/privateLinkScopes/deploy.bicep

```diff
@@ -59,7 +59,7 @@ resource privateLinkScope_lock 'Microsoft.Authorization/locks@2016-09-01' = if (
 }
 
 module privateLinkScope_privateEndpoints '.bicep/nested_privateEndpoint.bicep' = [for (endpoint, index) in privateEndpoints: {
-  name: '${uniqueString(deployment().name, location)}-Insights-PvtEndPnt-${index}'
+  name: '${uniqueString(deployment().name, location)}-PvtLinkScope-PrivateEndpoint-${index}'
```

Comment:
Should we always use `Pvt` for `Private?`

Reply:
Changed to `Private `as deployment name won't exceed max lenght